src/style.css

```diff
@@ -211,6 +211,19 @@ span {
   color: rgb(250, 53, 53);
 }
 
+/* Modal */
+
+.modal {
+  position: fixed;
+  background-color: #e0ce00;
+  padding: 10px;
+  z-index: 999999;
+  top: 0;
+  height: 100vh;
+  opacity: 0.9;
+  backdrop-filter: blur(4.5px);
+  display: flex;
+
```

Comment:
You forgot to close the brackets in this part. Also, we can add more style to the popup, like center the card on the page.

Reply:
Thanks for the review. But there's already a closed bracket. The card is also in the center. The element class used to apply the center property is called modal-popup not modal. You can check it